server/website/website/management/commands/stopcelery.py

```diff
@@ -11,7 +11,7 @@
 
 
 class Command(BaseCommand):
-    help = 'Start celery and celerybeat in the background.'
+    help = 'Stop celery and celerybeat and remove pid files.'
```

Comment:
Per our discussion yesterday I think this is the only change in this PR. Can you add this small fix to #252?

Reply:
OK. I will close this PR.